include/klee/Support/PrintContext.h

```diff
@@ -10,11 +10,11 @@
 #ifndef PRINTCONTEXT_H_
 #define PRINTCONTEXT_H_
 
-#include "klee/Expr.h"
 #include "llvm/Support/raw_ostream.h"
 #include <sstream>
 #include <string>
 #include <stack>
+#include "klee/Expr/Expr.h"
```

Comment:
Quoted includes should come before `<>` style includes. See http://llvm.org/docs/CodingStandards.html#include-style